### PR TITLE
CLXP-54 MVP Design Review and adjustement

### PR DESCRIPTION
**Pull request by HuyIT9898**
Purpose:  My learning MVP Design Review and adjustement

Review app: unable to create one cause out of snapshot allowed monthly for free account :(

preview: https://66224c8b94e874634f31de6b-fapxgxlxmo.chromatic.com/?path=/story/template-mylearning--default

### CODE REVIEW

File: packages/@coorpacademy-components/src/template/my-learning/index.js [hunk 1]
```diff
@@ -154,7 +154,10 @@ const MyLearning = (props, context) => {
   const graphDatas = useMemo(
     () =>
       pipe(
-        map(skill => [skill, skillsInformation[skill].stats.score]),
+        map(skill => [
+          skill,
+          skillsInformation[skill].stats ? skillsInformation[skill].stats.score : 0
```
Comment (AdrieanKhisbe) [inline packages/@coorpacademy-components/src/template/my-learning/index.js:159]:
```suggestion
          getOr(0, [skill, 'stats'], skillsInformation)
```

**Comment by wernerdurand** [comment]
Hi @HuyIT9898 Looks very good, but I noticed something weird with the search bar when using the suggestions, a weird light blue square appears
Capture d’écran 2024-07-02 à 18 13 24
 
 Everything else looks good!

**Review by AdrieanKhisbe — APPROVED** [review]

File: packages/@coorpacademy-components/src/template/my-learning/index.js (hunk 1 above)
Comment (adamska27) [reply packages/@coorpacademy-components/src/template/my-learning/index.js:159]: finally you remove the getOr, there was an issue with it?

**Review by adamska27 — APPROVED** [review]

File: packages/@coorpacademy-components/src/template/my-learning/index.js (hunk 1 above)
Comment (AdrieanKhisbe) [reply packages/@coorpacademy-components/src/template/my-learning/index.js:159]: please use it, that will avoid it to break if skill data or stats props isnt there for some entries.
(happened a lot to Adams and I while working on the backend, but not sure we supported all the cases)

Comment (AdrieanKhisbe) [reply packages/@coorpacademy-components/src/template/my-learning/index.js:159]: Original suggestion was wrong (partial path)
```suggestion
          getOr(0, [skill, 'stats', 'score'], skillsInformation)
```

**Pull request merged**